src/server.cc

```diff
@@ -248,20 +244,26 @@ void Server::shutdown()
 {
     int ret = 0; // return code
 
-    /*
-    ContentManager::getInstance()->shutdown();
-    UpdateManager::getInstance()->shutdown();
-    Storage::getInstance()->shutdown();
-    */
+#if 0 // enable later
```

Comment:
I guess you are addressing this shortly?

Reply:
yes, i was first confused by the existing commented code and did not want to add something new. but then i realized that the singleton was always calling shutdown.
i will enable this.